pkg/util/github/github_test.go

```diff
@@ -60,51 +59,64 @@ func testBody(t *testing.T, r *http.Request, want string) {
 	}
 }
 
-// setup sets up a test HTTP server along with a github.Client that is
+// Setup sets up a test HTTP server along with a github.Client that is
```

Comment:
Some functions are the same. Is it necessary to expose them?

Reply:
> Some functions are the same. Is it necessary to expose them?

I agree with you.